### PR TITLE
LPS-159839 | Add new test JSONCanBeConfigured

### PR DESCRIPTION
**Pull request by bronsonsoda**
**Background**
JSON can be configured on System Settings

**Test plan**
Given Walkthrough in System Settings
When JSON description is configured
Then success message appears

https://issues.liferay.com/browse/LPS-159839

### CODE REVIEW

**Comment by bronsonsoda** [comment]
ci:test:frontend:nocompile

**Comment by bronsonsoda** [comment]
This test seems to be the same as [CanBeEnabled](https://issues.liferay.com/browse/LPS-159838)

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase [hunk 1]
```diff
@@ -58,4 +89,68 @@ definition {
...
+
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1",
+				widgetName = "JS Walkthrough Sample");
+		}
```
[the remark this answers is not in the document]
Comment (susanchen3) [reply modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase:149]: No need to enable walkthrough, I just wanted to check that you can input JSON steps and save without enabling (check mark the box) Walkthrough.

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase [hunk 2]
```diff
@@ -19,6 +19,37 @@ definition {
...
+			Walkthrough.enableWalkthrough(fileName = "walkthrough_configuration.json");
+
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1",
+				widgetName = "JS Walkthrough Sample");
```
Comment (susanchen3) [inline modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase:45]: Same for here, I just want to check that you can check mark "Enable Walkthrough" without inputting JSON steps and save.

Comment (susanchen3) [reply modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase:45]: Although now that I think about it, does it make sense to have the option to enable walkthrough if no steps are provided? :thinking:

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase [hunk 3]
```diff
@@ -19,6 +19,37 @@ definition {
...
+		}
+
+		task ("TThen success message appears") {
```
Comment (susanchen3) [inline modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase:48]: Also typo here :P

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase (hunk 2 above)
Comment (bronsonsoda) [reply modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase:45]: [Fixed here ](https://github.com/liferay-frontend/liferay-portal/pull/2721)

**Comment by bronsonsoda** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
cc/ @bronsonsoda please see edits. Looks good to forward.

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**